scripts/zones/Attohwa_Chasm/mobs/Lioumere.lua

```diff
@@ -8,6 +8,30 @@ require("scripts/globals/missions")
 -----------------------------------
 local entity = {}
 
+entity.onMobWeaponSkill = function(player, mob, skill)
+    --Sets entire party enmity CE/VE to 0.
+    local alliance = player:getAlliance() -- Gets member pointers for alliance
+    for _, member in pairs(alliance) do
+        mob:resetEnmity(member) -- Resets player enmity on the mob
+    end
```

Comment:
Shouldn't this check for player pets aswell?

Reply:
Yeah you are right. I'll fool around with it tomorrow